src/pages/api/projects/index.page.ts

```diff
@@ -0,0 +1,84 @@
+import { queryDBProjects } from 'lib/api/supabase/projects'
+import { DBProjectQueryOpts } from 'models/dbProject'
+import { ProjectTag } from 'models/project-tags'
+import { NextApiHandler } from 'next'
+
+// Searches Juicebox projects matching text query param
+const handler: NextApiHandler = async (req, res) => {
+  const { text, tags, page, pageSize, archived, pv, orderDirection, orderBy } =
+    req.query
+
+  if (text && typeof text !== 'string') {
```

Comment:
solid validation here, nice work.

We have `yup` now that we can/should use for validation going forward. Non-blocking for this pr tho 🙏

Reply:
discussion raised: https://discord.com/channels/939317843059679252/939705688563810304/1093330486891401226